installer/tue-get-dep.bash

```diff
@@ -54,19 +55,19 @@ function _show_dep
         #echo output
         if [[ "$PLAIN" = "true" ]]
         then
-            echo $1
+            echo "$1"
         elif [[ "$VERBOSE" = "true" ]]
         then
-            echo $indent_str$1 $version
+            echo "$indent_str""$1" "$version"
```

Comment:
Only one pair of double quotes are sufficient and all variables can be defined in it